src/sentry/static/sentry/app/views/settings/project/projectFilters/projectFiltersSettings.tsx

```diff
@@ -68,7 +68,27 @@ const LEGACY_BROWSER_SUBFILTERS = {
 
 const LEGACY_BROWSER_KEYS = Object.keys(LEGACY_BROWSER_SUBFILTERS);
 
-class LegacyBrowserFilterRow extends React.Component {
+type FormFieldProps = React.ComponentProps<typeof FormField>;
+
+type RowProps = {
```

Comment:
shouldn't we name it `Props`?

Reply:
There are two components in this file, a row and container. I used `Props` and `State` for the container which is lower in the file (line 186)